### PR TITLE
Create UgradeTeamsUsingYeoman.md

### PR DESCRIPTION
**Pull request by divya-akula**
This would help the beginners to understand how to update / upgrade teams version to the latest

### CODE REVIEW

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+```
+ yo teams --version
+```
+## Update teams
+The yo command lists out various options ranging from creating project to upate generator , use the below command to select updtae generator
+```
+ yo
+```
+Use the arrow keys to choose Update Generators
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams/YoSelectUpdateGen.png)
```
Comment (KirtiPereira) [inline msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:13]: Kindly check and correct file paths, as they should not be pointing to personal folders.

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+This should listdown all the generators available , you should select a specific generator by using space bar to select and deselect teams from the available options
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams//UseSpaceToSelectGenerators.png)
```
Comment (KirtiPereira) [inline msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:16]: Kindly check and correct file paths, as they should not be pointing to personal folders.

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+It takes a while to install teams, may be a few seconds to minutes
+
+Once the installation is complete , you can use the below command to check the version
```
Comment (KirtiPereira) [inline msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:20]:
```suggestion
After the installation is complete, use the following command to check the installed version:
```

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+```
+yo teams --version
+```
```
Comment (KirtiPereira) [inline msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:24]: Same comment as for the previous command: Will we need to call out the language against the ticks?
For example, mention json after the (```) ?
```json
yo teams --version
```

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams/FindVersionAfterInstallation.png)
```
Comment (KirtiPereira) [inline msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:25]: Kindly check and correct file paths, as they should not be pointing to personal folders.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Kindly go through the review comments, verify and incorporate. Also, please ensure that the Validation warnings are resolved.
Thank you.

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# Upgrade teams using yeoman generator
+This article helps you to upgrade teams from current version to the latest version
```
[the remark this answers is not in the document]
Comment (divya-akula) [reply msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:2]: @KirtiPereira  Thank you will do

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md (hunk 5 above)
Comment (divya-akula) [reply msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:25]: @KirtiPereira  Sure as the folder is  approved I will add it.

Also need help in understanding this.
image

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+## Get current version of teams
+```
+ yo teams --version
+```
```
[the remark this answers is not in the document]
Comment (divya-akula) [reply msteams-platform/tutorials/UgradeTeamsUsingYeoman.md]: Updated to include language against snippets

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+## Update teams
+The yo command lists out various options ranging from creating project to upate generator , use the below command to select updtae generator
+```
+ yo
+```
```
[the remark this answers is not in the document]
Comment (divya-akula) [reply msteams-platform/tutorials/UgradeTeamsUsingYeoman.md]: Updated to include language against snippets. Thank you.

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md (hunk 5 above)
Comment (KirtiPereira) [reply msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:25]: For the validation warnings, you can refer to another document to get an idea.
For example:
Title : Upgrade Teams using Yeoman
Description: This document provides steps on how to upgrade Teams using Yeoman.
For the duplicate-alt-text message change the description little for each of the line mentioned.

Hope this helps.

File: msteams-platform/tutorials/UgradeTeamsUsingYeoman.md (hunk 1 above)
Comment (divya-akula) [reply msteams-platform/tutorials/UgradeTeamsUsingYeoman.md:13]: @KirtiPereira  it depends on the bandwidth

**Review by divya-akula — COMMENTED** [review]
Updated to include relative images

**Review by divya-akula — COMMENTED** [review]
Upgrade Microsoft Teams using the Yeoman generator - has been updated

**Comment by KirtiPereira** [comment]
@divya-akula - can you share the ship room feature number or any work item number associated with this PR? I will go through the document before merging. If there are no more changes required will merge the PR, else will inform you to make the changes. 
**Please note**: Currently, the builds are not generating due to a technical glitch, hence, merging of PRs is put on hold. We are following up with the team and will let you know once the issue is resolved, and the PR is ready to merge.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**